__tests__/app.test.js

```diff
@@ -42,4 +42,39 @@ describe('GET /api/topics', () => {
             expect(response.body.endpoints).toEqual(endpoints)
         })
     })
+})
+describe('GET /api/articles/:article_id',()=> {
+    it('GET: 200 responds with a article depending on the article_id', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then((response) => {
+            expect(response.body.article).toHaveProperty('title')
+            expect(response.body.article).toHaveProperty('author')
+            expect(response.body.article).toHaveProperty('topic')
+            expect(response.body.article).toHaveProperty('article_id')
+            expect(response.body.article).toHaveProperty('body')
+            expect(response.body.article).toHaveProperty('topic')
+            expect(response.body.article).toHaveProperty('created_at')
+            expect(response.body.article).toHaveProperty('votes')
+            expect(response.body.article).toHaveProperty('article_img_url')
+        })
```

Comment:
We need to be more specific here. We know which article we want so we can check everything about it including its property values.